src/schemas/chat.py

```
@@ -1,6 +1,20 @@
 # src/schemas/chat.py
 from pydantic import BaseModel
+from typing import List
```

Ruff (0.14.8)

`typing.List` is deprecated, use `list` instead

(UP035)